### ISSUE TITLE
Dataset segmentation

### ISSUE THREAD

**Issue by qa1234561** [CapWidow/W-Transformer#4]
I have read your paper and there are a few questions that have confused me. 
How did you segment the dataset in your paper first? As mentioned in your algorithm introduction, how do the h data in N+h come from?
How did you obtain the data compared in your test set?
Thanks！

**Comment by CapWidow** [comment]
Hi, 
if I understand your question properly here is my answer : at the beginning I split my dataset let say my forecast horizon is 12, I will take total datapoint - 12 for the training and 12 datapoint for the forecast horizon.  Then during the training the model will split the data kind of like batch which in the code reflect as  input_chunk_length , output_chunk_length. It will take for example  from 0 to 12 and predict 13, then from 1 to 13 and predict 14 ... During the testing I will from the training set, predict the test unseen value.
I hope I could describe it to answer your confusion.

**Comment by qa1234561** [comment]
Thank you for your answer. In your paper, I did not see the segmentation process between your test set and training set, so I do not quite understand how you decomposed the original data with a length of N into subsequences with a length of N. However, where did your [N+1: N+h] data come from. Is your h data sourced from the original data and evaluated against the h predicted values in the original data during the final testing.
